daemon/algod/api/server/v2/handlers.go

```diff
@@ -732,6 +732,54 @@ func (v2 *Handlers) GetBlockTxids(ctx echo.Context, round uint64) error {
 	return ctx.JSON(http.StatusOK, response)
 }
 
+func getLogsFromTxns(txns []transactions.SignedTxnWithAD, blockLogs *[]model.BlockLog, outerTxnID string) {
+
```

Comment:
You generally want to just return the slice, rather than pass a slice object in by reference. (Slices in Go are small, they pretty much _are_ a reference.)